### PR TITLE
fix: show only intercom icon on mobile version

### PR DESCRIPTION
**Pull request by Milosdanjov**
https://autoricardo.atlassian.net/browse/CAR-8372

### CODE REVIEW

**Comment by masone** [comment]
mobile/desktop hiding behavior works perfectly fine (tested in the listings PR)

The position of the icon is not nicely centered. the icon looks to have a larger distance to the right than to the left:
Screenshot 2021-09-21 at 09 17 24

There is also some (existing) issues with vertical center that I'd tackle while you're at it. the text seems to have a larger distance from the top than to the bottom:
Screenshot 2021-09-21 at 09 17 18

**Comment by Milosdanjov** [comment]
> mobile/desktop hiding behavior works perfectly fine (tested in the listings PR)
> 
> The position of the icon is not nicely centered. the icon looks to have a larger distance to the right than to the left:
> Screenshot 2021-09-21 at 09 17 18
> 
> There is also some (existing) issues with vertical center that I'd tackle while you're at it. the text seems to have a larger distance from the top than to the bottom:
> Screenshot 2021-09-21 at 09 17 24

Thanks, will fix that 👍

File: src/components/intercom/index.module.css [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
-  min-width: 123px;
-  padding-left: 13px;
+  padding-left: 18px;
   padding-right: 18px;
   z-index: 15;
 }
 
 @screen md {
   .launcher {
+    min-width: 123px;
```
Comment (masone) [inline src/components/intercom/index.module.css:13]: that's a pretty random number :D how does it come to be?

Comment (Milosdanjov) [reply src/components/intercom/index.module.css:13]: That number already existed, I didn't change anything..

**Comment by Averethel** [comment]
The mobile version looks fine now, but is it only me or does the desktop have more space on the left 
Screenshot 2021-09-22 at 09 36 05

**Review by masone — APPROVED** [review]
position looks great now! 💯

**Pull request merged**